Doc/library/asyncio-stream.rst

```diff
@@ -290,6 +290,10 @@ StreamReader
       Return ``True`` if the buffer is empty and :meth:`feed_eof`
       was called.
 
+   .. method:: close()
+
+      Invoke ``close()`` on the underlying transport (if one exists).
```

Comment:
it is not clear what "what" means here: "`close()` method" or "underlying transport"?

Reply:
A transport is a pervasive concept throughout the asyncio package. `StreamReader` objects usually have an internal transport member. I can add a link to the transport documentation page here to make that more clear. Or what do you suggest?